coral/src/app/features/topics/list/TopicCard.tsx

```diff
@@ -30,39 +31,50 @@ const exampleCard = {
 
 function TopicCard() {
   return (
-    <BorderBox
-      style={{
-        width: 400,
-      }}
-    >
-      <Grid
-        colGap={"l1"}
+    <li>
```

Comment:
Yeah that makes sense! I did go back and forth with that a bit and decided to do a `li`, but I think solving it the way you suggested is a good way to go! 

And that also made me aware that I have to update the card a bit because `li` only should have flow content as children 😱  and I threw `div`s in there.